04_procedury_a_funkce/4_procedury_fce.py

```diff
@@ -3,6 +3,21 @@
 
 Slouží ke zjednodušení kódu – podprogram se definuje jen jednou a následně se v programu už jen deklaruje jeho název (odkazuje se na něj) 
 
+Např. chceme získat n-tou mocninu x, nebudeme všude psát x * x * x * ... ale
+vytvoříme si funkci:
+
+int pow(int x, int n)
+{
+    int ret = 1;
+
+    while (n--)
```

Comment:
Možná by to chtělo ošetřit případ n<1, aby nedošlo k zacyklení, ale jde o příklad

Reply:
Jojo, to je pravda. Mocnina vlastně může být záporná, takže by tam mohl být pro tento případ ještě druhý branch, ale to už je až moc detailů pro něco co má vysvětlit použití funkcí 🥸